meta/main.yml

```diff
@@ -11,6 +11,13 @@ galaxy_info:
         - 16.04
         - 18.04
   galaxy_tags:
-    - Strongswan
-    - IPSEC
-dependencies: []
+    - strongswan
+    - ipsec
+dependencies:
+  - role: onaio.monit
+    vars:
+      monit_scripts: "{{ ipsec_monit_scripts }}"
+      monit_setup_mode: "{{ ipsec_monit_setup_mode }}"
```

Comment:
Please pin this to `false` so that in case this role is not the only role that tries to set up Monit on the same server, changes from the other roles aren't overwritten.

Reply:
Pinned it to false with this var `ipsec_monit_setup_mode`